zaza/openstack/charm_tests/manila/tests.py

```diff
@@ -85,6 +86,29 @@ def test_manila_api(self):
     def _list_shares(self):
         return self.manila_client.shares.list()
 
+    def test_902_nrpe_service_checks(self):
+        """Confirm that the NRPE service check files are created."""
+        units = zaza.model.get_units('manila')
+        services = ['apache2', 'haproxy', 'manila-scheduler', 'manila-data']
+
+        cmds = []
+        for check_name in services:
+            cmds.append(
+                'egrep -oh /usr/local.* /etc/nagios/nrpe.d/'
+                'check_{}.cfg'.format(check_name)
+            )
+
+        for attempt in tenacity.Retrying(
+            retry=tenacity.retry_if_result(lambda ret: ret is not None),
```

Comment:
I think this is wrong now that we have an `assertIsNone`. I.e. in case of failure, we won't have a None ret but a raised exception instead. I believe this should now be something like

```
        for attempt in tenacity.Retrying(
                wait=tenacity.wait_fixed(20),  # seconds
                stop=tenacity.stop_after_attempt(12),
                reraise=True):
            with attempt:
```

Note that I changed the `wait_fixed` as well, as waiting 2 minutes between retries was extremely long IMHO.